Adding a sample method for posting data

sfc-gh-eraigosa opened the pull request:
We need a method for getting cordinates so this helps us with that

Commit:
Initial commit

Commit:
Update README.md

Commit:
adding project description

Commit:
Update README.md

Commit:
adding some notes from this morning

Commit:
this is proto container for heroku geo api app

Ashley and Martin are working on api functions that will allow
us to work with geo api's in python. This is a dummy container
that will eventually host their app so we can connect it with
the facebook container app that lets us interact with immigrants.

Commit:
fixing app setup

jorgeluiso committed:
v0 messenger app

jorgeluiso committed:
Merge pull request #9 from jorgeluiso/messenger

v0 messenger app

Commit:
get local testing working

Commit:
correct heroku app deployment

Commit:
Merge pull request #8 from wenlock/wenlokc/working_proto_api_geo_app

this is proto container for heroku geo api app

Commit:
adding heroku docker wrapper for messenger-app

Commit:
set execution bit

Commit:
Merge pull request #10 from wenlock/wenlock/messenger_heroku

adding heroku docker wrapper for messenger-app

Commit:
Adding a sample method for posting data

We need a method for getting cordinates so this helps us with that

wenlock commented:
This is one for @yumibyte so that she can begin integrating her functions for geo location.

Commit:
These are notes from afternoon on day 1

Commit:
Merge pull request #15 from wenlock/some_notes

These are notes from afternoon on day 1

Commit:
This sets up our initial deployments

We should be deploying the master branch with travis for our apps
that we are creating. 
The `make build` target can decide to perform testing or not as well.

Commit:
Lets test out deployment

This test out deployment from any branch so we can work out issues in 
this branch

Commit:
Adding generating netrc file

Commit:
lets test login

Commit:
try a real deploy

Commit:
this makes it officially deploy on master

Commit:
quick update to the build status

Commit:
Merge pull request #17 from wenlock/wenlock/implement_deployment

This implements ci for our project

Commit:
Merge branch 'master' into wenlock/sample_json_post

wenlock commented:
TODO:

This should only be merged if we refactor it to :

- [ ] just add a ping template

Otherwise it's not needed.
